packages/browser-sdk/src/lib/embed/index.ts

```diff
@@ -80,6 +85,15 @@ interface WherebyEmbedElementEventMap {
     screenshare_toggle: CustomEvent<{ enabled: boolean }>;
     streaming_status_change: CustomEvent<{ status: string }>;
     connection_status_change: CustomEvent<{ status: "stable" | "unstable" }>;
+    precall_check_completed: CustomEvent<{
+        status: "passed" | "failed" | "skipped";
```

Comment:
```suggestion
        status: "passed" | "failed";
```
https://github.com/whereby/pwa/pull/4050#discussion_r1562507877

Reply:
Update in 8ce34600d85ad69e6dcde35a6faf9fac0884c494 👍